packages/typespec-test/test/openai_modular/generated/typespec-ts/src/rest/responses.ts

```diff
@@ -100,13 +100,6 @@ export interface GetAzureBatchImageGenerationOperationStatusDefaultResponse
     GetAzureBatchImageGenerationOperationStatusDefaultHeaders;
 }
 
-/** The final response for long-running getAzureBatchImageGenerationOperationStatus operation */
-export interface GetAzureBatchImageGenerationOperationStatusLogicalResponse
```

Comment:
The get polling operation won't be taken as lro in getLroMetadata, so relevant logic response will be removed.

So I think this change is expected.